### DISCUSSION TITLE
Import peers no active

### DISCUSSION THREAD

**Discussion by xgusto** [eduardogsilva/wireguard_webadmin#17] [Q&A] [answered]
I want to ask if it will be possible to import a peer from a file in the peer section in the newer version.
Now the button is inactive.

**Comment by eduardogsilva** [comment, marked as answer]
Hello @xgusto I've created an issue regarding the import tool. It will be implemented soon.
You can discuss/follow the issue here: https://github.com/eduardogsilva/wireguard_webadmin/issues/18 

Cheers!
